virttest/shared/cfg/base.cfg

```diff
@@ -1032,6 +1032,25 @@ uuid_dimm = ""
 # qemu_cmdline_format_cfg = file:/home/user/libvirt-latest.json
 # qemu_cmdline_format_cfg = string:{"images":{"blockdev": "json"}}
 
+# The key 'image_iothread_vq_mapping" in QDevice.
+# This key may have many structures of value.
+# Example:
+# iothread_vq_mapping = x:0,1,2 y:3
+# iothread_vq_mapping = x y z
+
+# There are some conflict scenes due to the fact that setting the inappropriate
+# parameters. The function conflict_check() is designed to detect the conflict.
+# The following scenes cause the conflict:
+# Scene#1: If iothread_scheme and image_iothread_vq_mapping both are set.
+#          If you want image_iothread_vq_mapping working well, suggest set
+#          iothread_scheme = “”
+# Scene#2: If iothread_scheme is set to multipeerroundrobin or full and
```

Comment:
Let's introduce these two new policies briefly in base.cfg

Reply:
> Let's introduce these two new policies briefly in base.cfg

The two policies you mentioned are: from line 1035 to line 1052 ?

Reply:
Sorry I meant the new schemes: multipeerroundrobin and full